### PR TITLE
feat: Add gpb-v3 subproject

### PR DESCRIPTION
**Pull request by jakubjanecek**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -43,3 +43,12 @@ lazy val gpb = project
     name := "bytes-gpb",
     libraryDependencies += "com.google.protobuf" % "protobuf-java" % "2.6.1"
   )
+
+lazy val gpbv3 = project
+  .in(file("gpb-v3"))
```
Comment (augi) [inline build.sbt:48]: Cannot we use here just `gpb`, and so don't duplicate the Java code? 🤔

**Review by augi — CHANGES_REQUESTED** [review]

File: build.sbt (hunk 1 above)
Comment (jakubjanecek) [reply build.sbt:48]: No, that's not possible. I agree that the code duplication is not nice but consider that this code hasn't been change in years and it probably won't be in the future so let's not overcomplicate this.

**Comment by sideeffffect** [comment]
If you want to "reuse" code from different module, you can do it like this https://github.com/keynmol/portable-scala-reflect-repro-1/blob/4b896f0ede237e0d5d063e1f935db41685e1a350/build.sbt#L21-L23
`unmanagedSourceDirectories` is the important part (there should also be an analogous key for test code).

**Comment by jakubjanecek** [comment]
OK, thanks, that seems to work well.

**Review by augi — APPROVED** [review]

**Pull request merged**